tests/subscribe.test.tsx

```diff
@@ -45,8 +45,10 @@ describe('subscribe', () => {
     expect(() => subscribe(obj.count, handler)).toThrow()
   })
 
-  it.skip('should not re-run subscription if no change', async () => {
+  it('should not re-run subscription if no change', async () => {
     const obj = proxy({ count: 0 })
+    await Promise.resolve()
```

Comment:
I would like to discuss this line, right now the `subscribe` handler is called immediately if you subscribe in the same tick as creating the proxy, but it is not called immediately if you subscribe in the next tick. I feel like either behaviour is fine as long as it's consistent. What do you think?

Reply:
Here is an example of a situation where the subscribe handler is called and it might even be desired in this case:
https://codesandbox.io/s/react-forked-p9dy0?file=/src/App.js

However if by default the subscribe handles are not called directly the user could manually call it if desired.

Reply:
> I feel like either behaviour is fine as long as it's consistent. What do you think?

Yeah, that would be less confusing.

When I added this L50 in the test, I thought it's unfortunate too.
I mean we should avoid notifying updates in `proxy()`. Maybe, adding `initializing` flag would help.

Reply:
c049b42 should be better.

Reply:
looks good!

Reply:
thanks!